patroni/ha.py

```diff
@@ -69,7 +70,7 @@ def touch_member(self):
             'role': self.state_handler.role
         }
         if self.patroni.tags:
-            data['tags'] = self.patroni.tags
+            data['tags'] = self.get_effective_tags()
```

Comment:
If statement is wrong. `patroni.tags` can be empty but 'nosync' set dynamically.